sdk/appconfiguration/app-configuration/src/internal/cryptoHelpers.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { createHash, createHmac } from "crypto";
+
+/**
+ * @internal
+ * @ignore
+ */
+export function sha256Digest(body: any): Promise<string> {
+  return Promise.resolve(
+    createHash("sha256")
+      .update(body || "")
+      .digest("base64")
+  );
+}
+
+/**
+ * @internal
+ * @ignore
+ */
+export function sha256Hmac(secret: string, stringToSign: string): Promise<string> {
```

Comment:
why not make this and `sha256Digest` async functions so you don't have to call Promise.resolve yourself?

Reply:
That seems totally reasonable :)

Reply:
Done.